python/mxnet/numpy/multiarray.py

```diff
@@ -9684,3 +9684,111 @@ def pad(x, pad_width=None, mode="constant", **kwargs): # pylint: disable=too-man
            [10, 10, 10, 10, 10, 10, 10]])
     """
     return _mx_nd_np.pad(x, pad_width, mode, **kwargs)
+
+
+@set_module('mxnet.numpy')
+def cross(a, b, axisa=-1, axisb=-1, axisc=-1, axis=None, **kwargs): # pylint: disable=too-many-arguments
```

Comment:
move the wrapper to after `outer` in the file to keep consistency. Same for all python wrappers